filter/etc/e-smith/templates/etc/pam.d/rspamd/10base

```diff
@@ -0,0 +1,16 @@
+{
+my @users = split (',', ($admins{'user'} || 'admin'));
```

Comment:
I don't remember these prop are defined as CSV list. Their name is a singular, not plural...

 Where did you find the documentation?

I'd prefer a single value syntax!

Reply:
[root@ns7dev5 ~]# config show admins 
admins=configuration
    group=domain admins
    user=admin

Reply:
Ok "admins" is plural :) ...because a group of users and a user  form together a plurality